README.md

```diff
@@ -138,3 +138,45 @@ authorised_user:
 
 The email address has to exist in the users table of manage-courses-backend, but
 the password can be any non-secure local password you care to use.
+
+## Cypress
+
+1. Configurations file
+   1. A `./end-to-end-tests/config/example.json` is available as a basis to create `./end-to-end-tests/config/local.json`
+   1. Change the values where appropriate
+
+### Custom browser
+1. Download the appropriate version browser from [Chromium Downloads Tool](https://chromium.cypress.io/).
+    1. Extract content to `./end-to-end-tests/browsers`
+
+### Executing tests
+
+1. To open cypress
+    ``` bash
+    # using ~/repos/dfe/manage-courses-frontend/end-to-end-tests/config/local.json
+    yarn run cy:open --browser $PWD/end-to-end-tests/browsers/chrome-linux/chrome
```

Comment:
I say we ditch the whole `--browser` thing. As mentioned above, it's more complicated on OSX and really, should be unnecessary since Cypress already discovers what browsers are installed.

Reply:
removed